compiler/one-cmds/tests/onecc_043.test

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Test for "O1=True" option in onecc config file
+
+filename_ext="$(basename -- $0)"
+filename="${filename_ext%.*}"
+
+trap_err_onexit()
+{
+  echo "${filename_ext} FAILED"
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+configfile="onecc_043.cfg"
+outputfile="inception_v3.onecc_043.opt.circle"
+
+rm -f ${filename}.log
+rm -rf ${outputfile}
+
+# run test
+onecc -C ${configfile} > ${filename}.log 2>&1
+
+if [[ ! -s "${outputfile}" ]]; then
+  trap_err_onexit
+fi
+
+readarray -t O1_OPTS < "O1.list"
```

Comment:
Q) is `O1.list` a file? where can I find this?

Reply:
This file dynamically exported during the build. 

```cmake
add_custom_command(OUTPUT ${O1_TXT_FILE_BIN}
  COMMAND ${PYTHON_EXECUTABLE} ${CONSTANT_EXPORTING_SCRIPT} --constant ${O1_OPTION}
                                    --format txt --output_path ${O1_TXT_FILE_BIN}
  DEPENDS ${CONSTANT_EXPORTING_SCRIPT}
  COMMENT "Generate ${O1_TXT_FILE}"
)
```

You can create it in your local PC with below command.
```bash
$ python3 compiler/one-cmds/onelib/export_constant.py --constant O1 --format txt --output_path O1.list
```

Reply:
OK, adding a comment about this just above this line will help understand better :)